include/up-cpp/utils/Expected.h

```diff
@@ -22,87 +23,161 @@ namespace uprotocol::utils {
+	// il, Args&&... args)
+
+	constexpr const E& error() const& noexcept { return std::move(storage); }
+
+	// this one is checked
+	constexpr E& error() & noexcept { return storage; }
+
+	constexpr const E&& error() const&& noexcept { return std::move(storage); }
+
+	constexpr E&& error() && noexcept { return std::move(storage); }
+};
 
 /// @brief A stripped-down version of std::expected from C++23.
-template <typename T, typename E>
+template <typename E, typename U>
 struct Expected {
-	using value_type = T;
-	using error_type = E;
-
-	constexpr Expected();
-	constexpr Expected(const Expected& other);
-	constexpr Expected(Expected&& other) noexcept;
-
-	constexpr explicit Expected(T&& v);
-	constexpr explicit Expected(E&& e);
-
-	Expected& operator=(const Expected& other);
-	Expected& operator=(Expected&& other);
-
-	Expected& operator=(const T& other);
-	Expected& operator=(T&& other);
-
-	Expected& operator=(const E& other);
-	Expected& operator=(E&& other);
-
-	constexpr explicit operator bool() const noexcept;
-	constexpr bool has_value() const noexcept;
-
-	constexpr T& value() &;
-	constexpr const T& value() const&;
-	constexpr T&& value() &&;
-	constexpr const T&& value() const&&;
-
-	constexpr const E& error() const& noexcept;
-	constexpr E& error() & noexcept;
-	constexpr const E&& error() const&& noexcept;
-	constexpr E&& error() && noexcept;
-
-	constexpr T value_or(T&& default_value) const&;
-	constexpr T value_or(T&& default_value) &&;
-
-	template <typename F>
-	constexpr auto and_then(F&& f) &;
-	template <typename F>
-	constexpr auto and_then(F&& f) const&;
-	template <typename F>
-	constexpr auto and_then(F&& f) &&;
-	template <typename F>
-	constexpr auto and_then(F&& f) const&&;
-
-	template <typename F>
-	constexpr auto or_else(F&& f) &;
-	template <typename F>
-	constexpr auto or_else(F&& f) const&;
-	template <typename F>
-	constexpr auto or_else(F&& f) &&;
-	template <typename F>
-	constexpr auto or_else(F&& f) const&&;
+	std::variant<E, Unexpected<U>> storage;
+
+public:
+	constexpr Expected() { storage = E(); }
+	constexpr Expected(E&& rhs) : storage(std::move(rhs)) {}
+	constexpr Expected(Unexpected<U>&& rhs) : storage(std::move(rhs)) {}
+
+	constexpr Expected(const Unexpected<U>& rhs) {
+		std::get<Unexpected<U>>(storage).storage = std::move(rhs);
+	}
+
+	template <class X = E>
+	constexpr explicit Expected(X&& rhs) {
+		std::get<E>(storage) = std::forward<X>(rhs);
+	}
+
+	template <class X = U>
+	constexpr explicit Expected(Unexpected<X>&& rhs) {
+		std::get<E>(storage) = rhs.error();
+	}
+
+	constexpr Expected(const Expected& rhs) { storage = rhs.storage; }
+
+	constexpr bool has_value() const noexcept {
+		return std::holds_alternative<E>(storage);
+	}
+
+	constexpr explicit operator bool() const noexcept {
+		return std::holds_alternative<E>(storage);
+	}
+
+	template <class X>
+	constexpr E value_or(X&& v) const& noexcept {
+		return has_value() ? std::get<E>(storage) : v;
```

Comment:
Based on the [cppreference page](https://en.cppreference.com/w/cpp/utility/expected/value_or) for this function, I'm pretty sure that the `v` needs to be forwarded:
```suggestion
		return has_value() ? std::get<E>(storage) : static_cast<E>(std::forward<X>(v));
```

Reply:
I added a test case for value_or. I tried the requested change but it does not compile. Removing the static_cast allows it to compile. I think there are other casts where static_cast could relax the return object initialization a bit, but it would require some focused attention. Not sure if it should be pursued now. I would recommend it should not.

Reply:
[It appears to compile for me](https://godbolt.org/#z:OYLghAFBqd5QCxAYwPYBMCmBRdBLAF1QCcAaPECAMzwBtMA7AQwFtMQByARg9KtQYEAysib0QXACx8BBAKoBnTAAUAHpwAMvAFYTStJg1DIApACYAQuYukl9ZATwDKjdAGFUtAK4sGEgOykrgAyeAyYAHI%2BAEaYxBIaXKQADqgKhE4MHt6%2BASlpGQKh4VEssfFcibaY9o4CQgRMxATZPn5cgXaYDpkNTQTFkTFxCUkKjc2tuR22EwNhQ2UjlVwAlLaoXsTI7BwA9HsA1ELKACIAGgC0AGJ0mB7JAJ7EeMAIBAAqmKoEIIdmGjMkkOHkEL2iXiIxAUhyIsIQmEO2GQtDwySUh2umwY6CYdQYJg0AEEDoSSUchJhEQQEYcIgB5D4ASTc2EONHoEAUq0O%2BHG4MhmHQhwA7oQEPC8DCRSQANbskiHJjofD4sRkg6HML8YgsPGZQ7ETDAJr4IyHNBPF5vAiHVAi8LQhBogB0Gr27sOH2dMOSxFQwGIrCVOPhiKYyDQLGShkeYWAhz1BDieDEMKaiL1WCVADcmHQmNF6IcvDi4mHPcndTDUFQw4ciTHkLTQjsGBiAGpxdICf4ujSi53NrXpvMFovhgie94EdEgA4ixcuphNhEukjAPaottKBR7YIs7ARITYS5mfvuz0nC6XVuMJSXJlYQR4GgjBurzBni/EsnmADMr5lnWcjKAA%2Bm4yjgXIzLBEIYHYOcyjYG4HzYKcYEABJ/mY/5YDQ4SHKBEFQWBMFMnBCFIShaEYdhv6/rhYQol42YmP%2BbjjFgqg7MkU7/tgOGAQwLFsRxBCPMkmBgQQQaEAo7GCYxwmiYi7FuHmLyGPxSlEmSzBsAoTaIl4fqoEQaC0CAICQnQMImP4VgMeSDaHEoL7hLQhyylSyTxqKeLDjqg6MKKiIKGKBDDnCbjWNYuEhsKJphEqkaYAoMJEJWtJGmIbkEOg1nfFJDhCi6hz0lshyoiwhBChaqBPG5CCbLQwqxIamDJAYOzCngLBsPgeI1I8brEuM%2BrIGBTAZXEBAQGAYBgWBCAzWByDJMk00ELJeAQsmEDLRtW2otEYHFd0yboOsZKHHd90PQ95hmAASl1PUmWZFmeNZtm0Ao1nYKoJVXaK4r5YVIAXaV6DPas7FOXpxIHEcAACBmIl80YkE0jxuV40TjIQgoKsQENFcDl1Cu6aNGnqxCyjClLUjlmBUHEjA7KT5NQ5TMMgDTj2PTOc4HIwLrHUa7NGiJmAS6gLB7CKezHXsf2EI8ezQ1dgtC/dESoOyWw0uW6CoMgPiMI0%2BIjocZk5ngWB9QwkowhycuC6jDmI/yXgOIcFjKkDINCkSaUZYcfzJAT2488QpaOGw53EP6ZPe3%2BGgAJyB%2BgwdU%2BgYc7BlEA8n8XHWfHL5J3EqcQM9wCMCmyAB0HfNXYX6UKWYZg8un/inBnSOZ8m0YGMmhzqRJUkYy6s8NsQwAKQJGfZ63IcF%2BHCgQESC9dwAbOYe%2Bz%2BVTSL/DDGZw9ZcFRXCf9dJNckFyN8gDqIqmupO%2BL4pECn9yx%2B90cg5Aev5%2B4I0HijQ4qNogvDZocN6ABHLweAjTCkaMAYA/kp6ImCqIXcIUjSHG1vVQwwpSzELQZJTAgsMxamfI4UQtAxpEhHt1YaE9xJUIxkiRSZIUQzRhHIcIbd6rp2JNHIseBkACwvmgdsyZgZkyERQiAcjxhEWEevQ%2Bvd/ynF5GzJgXhaD8URoSTOaiFF%2Bg0So5RIjYZmAPg4nRej8KGOMeA5yZiLHFTJsVbchBrF2NUQIdRgkHGGgQNySO%2BUcYNwgMQSJgCrBgNkSEyxvjgb%2BNtLY9eEAwmOL3hEqJ19YmYGfpDFgqAcxlISdyJJwDB5mNYWPNSHF%2BEZWPvPb%2By9UnyJ8UQzJUiAk5PzuU6yYQwJsJ2DJUgXT96H06X/c%2BQ8r4xKDHE8ur8SDv2ILDDiX8l7YF/rvVYACJ5AJSUjIe3jFENXkTw8Jj9iAlzueMQ%2BhwGCoG%2BLxW03tOoEC2C7cYpSEbnJAVcrxaT%2BkWIeQUohKcn48gsYfd5nzvldV%2BY5f5gK1lMAbqChpnis43Ksfk95TyXkovCWiniGLzkWGxcQIFL9KnVOfqU%2BG/4GWEquX6PAeZkwyKHmyYF6zqFcr/GAiVzlIHQNgXWIk%2BUXibSFJcM2DpDjVOhAaWsPMKHsn9CwEEcVLC4WYc09hk8uGsExrM7B3DBI9LDgYCOecYb0rJBI7cQqAKvkOBoFeJKyZuquhAD4VLCm1NLrijZLKqk1MSfUy5kK%2Bm3JDUKCAIyYbqUdWS8JUbomirxWUzZrKE11I9cm3p4x%2BnpvQME%2B5WadYcVze8gtYih5mM2Q3fibgm3UxbbwgS7KxWnKLQ3DhejS3xviYkjxnaq1DwtePdS7SYTnEnV6Xh1b0kDO6kM20daIDnAjUU%2BpF8u0vx7epcNw7x2YGcTzN%2BH8OInuHVG%2BdZieUr2Xa0twa7DgbvYno1tO7%2Bl%2BIPUiIJ/a9luDfXmyNiSPUXqzt2zAvbc0juLY%2B2pLoKWctMVnb9YG01BJhXWttSGSlitnQoF096k3goAjUJQt17q/qtdPG1nSDnbsVfdINUHcm8YcQsueSzC1QmLWMrZxAdmwd48OpZZy%2B5Mdwq4V8jTiVQtudEVAnhDirQUGBPM3gylIrSR8r5tK%2BLIc7VnI0AKmU8xam1YzYgqzMEcNUm9P8GOfqI4ulNNbbkQeQAExqcQ8SKj054F5MKaU/Ls1%2Bhz6GcWbNc%2Bgdzxi4hef5eK1CfmpMNwIyvYjQ9NS/o4f%2Bl166%2BNmM1IJj4mqxBeGkk/E9onwk5gs/I1F1mksdoa0cMxjmcVGZM618zHDrg82vRxW9RyGORxXpqCaDD1ozQw0VyGz7dnqXgxAHrpWL6anK00zAo9LVtNq4B%2Br2nU1WOa6ZtrYEOunp668qc1KBt0qG0R1LTmXYTZe9N9is20O9sW1hkrknJqbbeQtnb1k9uwcO8dgLmdzsPZC1YmF4butTfi2k95/2sdZz9QtMAIOierGWSl8nw8ED%2BhFC3XOdiO7FzKwDy%2B91nosNYbaOEEYi4wlBy8kUCIXbkLsW6buJ37OZzG85yHvm73FYfZj7H5idNPZa2ZylXXClk7MZTxaNODd0%2B54zmkLO2d1s51va3ZinrdwF5d2zwvN767axLqXJZNH5zlz3LXgP0txrZarpH6uOUK6/UFnHu7yPkoRc83rbzwkm4p3WCAFvfdW5Qzb5n9p7cc83nXQvLu%2Bdu52h7oXhsRed3hbXHkkvQqfNtDL9ewe488%2BV8yyG82%2B2B%2BzYO5eMfR14dTyXLXCedePeDc3xFE8jfJZ55TvP5n6c86acX1nOcHfl%2Bd1nV3Zh3fRnr6lUXS%2B08hRdh3gPFCe%2Bh6V2llXEeS1XvQ%2BpGDOah1LY1zpynxb1nyYxIzx0swJ0KUiyDChAACpid7ks9M4zdqc1pxcC9Fdd87cD8y9RcK8sCT9q8z9a8L9YRDYsApYOZZY9184fdEQ29pcR8rpn9pVFd%2B85tv9o8ACOVQCtN59ccyZ8c4C7QpJYCSBLgf509MVCNGdUDN8S5t9GdsCS9cD15HcCCGcq87p%2BdSDPcKCOZpZOZER9VUgwgqw79H9Zc4YX8ODD4o9CsJ9sM%2BDnI%2BUBV2AV5NlNJUxBAb1Zlf8x9c0Y1xVCMh51spFppZpmgqdNkpQTNUBHYTM1dsBSAV49Y9ZnoAB1REM2BgMADgW0MQWgEvR1AeXRHMBI4UCAUsVEHyPVOxeGeXALMI62CIgROaTQogu6WI4zLAX2HoCcJI7glfOFRaHo6aFOJgR4IYxwtkCNNIvWMYl%2BOIqgmWaZHMZI1Ii%2BU/OtTqJBFBdKJEcgxMKkIXWkWKSwawQ4fIvo2SP2RoCcfIvY5BOmK2GwtghpD45GTUL2ILfuO6TUDGIyCMT6f0b6KyGyRwf6RpdTHEP1Q4TUYiSCaCWCeCRCZCVCdCLCCBCkM4K4W4egB4Z4V4d4L4H4P4AEIEEEWQAUKETKQ2E2JEFENEDELEUsXEfES8b444KkesBkZkVkdkO4LkHkPkXafaeqSKCUGkKUUUOUbmZUVUTIdUbk7UEgJMA0I0E0XZfyS0Ykm0O0B0bsZ0ZIZhUkbk70WUsyQMYMUhesEXBWGMBgOMc0JMFMNMJUQhLMcMMcAwCcAPSgisbkqsFgGsOsRkxsCMFsKRe8RELsbVXsc8AcSXKRCUWUpgX0wsYsPEacHaUWJWJcFcKMuWDcLcGM9sdKfcQ8Y8U8JMrklya8K4O8Csx8ehV8PAd8SM5sL8OszxJiESbwMSNwYAZMcYPYEc9KAgF0BAe7PSfs1SarIMHEBWWcoE4yNfIkLwdIc0UyME82H6SEuyayNQ/OR3edLc/yXc8yfciEv6AGEACjNgi8ncr6G836KE%2B8gI0IgDOtL4dRKOGOKRWESc%2BMayP8mQz1ME/OH1LOTUF6UsO0GgsQr0bAIQD4MCa4ZhYbIiJQNBQ2JQTvZIC0egQwIhBgB2f0BgNgQQX0csUcqcleCox2Hk%2BQZIF5eNFOR2NSC5MAzOJi4UL4JoU4e0BgdirVF4NiHi/guChCgQLmZC74boWyXsXVRk%2BipUTabcfUAQLC2Co4RQeqYKAi0yO0OsIo4C8YOjQ4TCGaK/JvOEGUmEMICaWWXSzOX8ycylXRfRKgNxExFeAAPw8vGC8pcQMSMX8qJUzhkpdjksRAUtUCUptlUtpHUpXH3VEHxDcsRNwsQtoDxmCmAGKOiDymMqIs2AIHSGzFUsnLo08NaObn4pYrkGSHAqEGQQOkY3qsmk1UqK9EwCEpErao6rByktATKNCOQUED3kkBkkOB7TAiXPrXPRaJ6s2SWoVjAiwAdi5n2y%2BMznCOblLQIC4Ezkzn/H8ETHml2UUMx0Op5lLFfHVLAnMK2qlHFOUoJA4nMMUl5HeogA0FmS4Gq3UkOH/A0F71fyBz%2BpCpYAIDj0XTQjQowryTsXAtmRDREDECaCxoMGIBWtN0EDclYG6laT0QWqWpn32sMThFoCoBdmAwngAFYrBRMmbTgXlJCBIhN851JzDZkfrua/kOCFASb6ACUpVZCabDZ9VGa6bRLIb0SaIwIPgXo5AjlYtaAIAKErd9qlbUIVa1ajkn8FDdbZD9b0LsAABFLkMWzAWZJ/DAxW6iA262220ee2w4OAnWz9RG1C9C64VG9edGwJEGXGnGxhJoAminIm0Wj2zdCm0hKmqWyEQ2eWzdEwFmlFdmzm36x8twfmrUXwoWrFDgmDd20mgjMFTHaW2g91OW%2Bm5OleC2jCokOCDW/TLWnW52jEgOtuk8bW2XU2nu5Wt2uO0mh22XfDBGsAokJGgOoO/OEOzGyO/GtfFA2Ou2hO9DRapOyG2u9OxmzO1mg%2BHOnkLmtkfOwuzZfkeMX64Wt/IFO2iWialeWu2W7y%2BWpui%2BFu1W9WiATWwe9eM25ul29Cv%2B42oe9A2nEe12m28e%2BgSe7vJ2zHFuseu22Zb2ho322e%2BelG4Kggfw5g8Otesncw4m%2BOxmxOnEb%2BoeA%2B%2BmjOrOtmjm8%2BvOuxdSG%2B3aIwAW4uuYoBQvMu5gjNBBzXfaz4lOuED%2BvRL%2B2Bvu9ugBzuoB/OEBn%2BsB1u%2BRk26By3WRhCeBjBuulg6enBwePBwOghoh7WDsKbekUh/hoechkR7eggXemh/e1O6qBho%2Bph0%2Blhw4C%2Bnm0fAuwQWZThl4Iwe%2BuxhnQRlRUJ%2BMOubuBEWgYouGSG8Rt%2B9xqRjxhW1BtR64fujuuLbunJ3u9RgezR4zFBvWtR9Bj2pBoPUHN7Z5ERlRvSRdfo20ZQfMNOSJmO20VQTHchx4ZosxTplBCAch1QHh20R4aNVQCAVQdYQ4R4CAGZytJjSWkx/2/BtGycjGtuIRPAJBB9de2u1QJxlx5amu9xvGKhneym3vOh9xw%2B7y4%2B7O3x/x/OzZR6o5yZWSdSUZ4gRSEJl%2BW%2B8Jku2Q0bR%2BnmPUHyMCb5trf5rpn%2BSZpZ1JjZi%2Bd%2BuxTdGR4p5WiBhRwphonR/F8pybbRq5uEIixm6dNlHWl0DA3FuB%2BZ2ZZIC%2BhZxly2m2x4Fli%2BmZ4x5yUxxemGEOptA5o56OzOU585%2B5ilw2G57y6hy5r4x52mzxl57xzO95thrRb64Jh6hgQ517PiQFjiAF//deyF6G8u0tJgWF%2BFgrM14dFFmZtF1%2BjFjJrFhu7JqpkpvJ%2BRwBopn15Wv1spqBipmB2Vu2TdGlspOl/DDl3R5lu2NlnRt27l5NxSPlj4v25GsxnZ8YPZsO1aVBCVqV255xmV6m656Vve4ZrOehhm9Vk%2BzV3O7mz5kF4toUX5k1yCJFgSYFyGUF4ACJiFsPd/CpW16SBQTt2DR1o5Z111yNzJnFoNg2/FgNolhNklsNsl/PVNm2lFx22nFNhNtNupmGel49zN/dpNiALB4Bk91dzllZ2ZO9nW3lmezZ3NoVq6EV4hmd0t9xs58ti52hsxWu%2BV8mu52t5V8Dp5tVvRV55h1ty%2B9h3VwhnmadjMdAbtxFlBc15A6JoJG12FrD1BPDk1%2Bd2ZF1vgqtyRz1z%2BxunRkNgpruzdp90pyB7vYe09g989wx6fVYR9826prl/jsqfDD9/lq5QV8xgJ4QO4hwQDuEYDhV6D1xyNyD%2Ba9TpV0IuD1VxtxDjVs%2Bvx7V3m01rpgd6yIdkdxlF2AFpNrN7ldFlVmWhj6Rpjrdo2gltj4B4l7z0lypkTkpt2w92XDAl0dljjs9gxsqCLpz0BkLvj2L9AYTxL0esTlLqT7N3BrZvN4O3Z0Oy6BoeOJTk5oDmtjTujuVyr3T/ghtxh5tkzj5tDoJjDud%2BlOzormGCABzhdqutJ91%2Bj9ebFzzjjljnzpRmGFpsxFuibwLiN6L5Lp/fDSLm99NlbwTl0BL8awedpw4AF04Sc0riCi%2BCZgZomoZ/a%2B68h5AHDtABOOtzOQ747%2B48ZomlFwZ2Z%2BZxZ5Z1Zv5O79abEb7VmqwLlauqKl79phtdRKHxT77QpU%2Bb70%2BNb6j45YAbbpJC0e74Hk1MH5z2esxAKuHk7ylLFQHh7wQS4LmgnyVN1q5chkn%2B46yCn4HzdANZV2T/NjDzG%2BHo79piVpnhwFnnHhOdn2fUdyVirkDytyXiD2rsDwvBrrxprrVtt1roXjD2JsFvhyXy1nFTXxz1JwLenhnTFkbr1xXxnNBm2pIA7rp/n%2BHkXoHhOY39ytR9dxRwNyX3%2BgLndoL63G3pNo9y3Nbl/IPjb8L2nTH8P0T4PuxNLwvCP8T1L69lw4VOPwG%2B3lBR3k753yn%2BG6TskLngrgt7rhTk73P%2B4wXh3174XlAUXomxmjniFnpxnMttTitmDuX6tmX7v/g/TtOhD5m1XlD%2BTgdNr2ZTXkd4/KGnFcuw3/rl/QbxXc3ugy3t3oPu3zX/P4Hzf3J/Jyb73wPg/jR/3xbn3uPsL7vVbqLy/pLl9lL4AxFGPsRqtd3h/rPnfhvl3wQT9gVvLj%2ByFDL0bM%2BIKxmZjcAIhkAPkdAJkSlxMgFA2AVPMpyH6GcR%2BbzMflfT1ba9h24La3ER1yQ4D4mZgRJsk3lyLtYOWcX%2BphBej0hMinRXnDoTb7aFYQxAG5swJ5xPQxGnA%2B6Gv3rqMdvWevLokiFP4D0N2fnZfsILm6H8Fu5LbgYzgei11Qcm6EPvnhfxPR/imVYcDD1tAnkYYZ5R5AswtY8C7oLdNCi9Ddq6FBcxxRvBHAwKWEu8QeZ6FsUIKMD0iQsb4C6Elx4hbq8glgbbntDqC%2Bc7/DQacBcEsC9B7cTeP/xk6AC5OQMH5JkCQGpxIB3QGAXAMYAGwCAyQkgCgKyaNcMBrDdXjq0n6YcuGuA3Xl1yZov0nu1A2gfQIWJMDJeD0WSOwKEFuDJBbgvgaDA36dCHovvf%2BuIOUZu8WBAwrjkHh45%2BDhB79FOCoKnqCcghTAvRFoIlA6DS86hTeOSiMHIFRhajcwZYJrzWCvc1%2BfDJYQfyOCL2zg2fu4PSKeDvB80GbjvmEEBCRQiwiHismCFhDGh6w08tEKL7EgS%2BS9QrgkIxSZAjuUscAW1gyEMAYMeQ55kZ1H5FDUOJQmFACys4gAbOeAgRlC2tYgtyhxA0gagBSa0c9OVAj3jQLoEMCNBzQ9jGwOMEKC%2BcUwroR6wt4CCreDOfoaINY5Tcrojw5QlIK5E8iyokw9oQ9BhQd89E97czjSKWEWhAoqwmFJENDibDDBErDkfdDMGq1LBleYQafnPz6F9EqxYwil3oIOChG6AYPOEKeFuCHodw1aA8JGHPC98bw4jB0P7hWi3BSojeKLhiHF84h3PPZokIEDgi2YygWSNCNhHlcDOBQ5DkiPH6ztLOZQsJhUPpEEDRkRA56ISOJFv9TeqjEph8ApENDtiwQmUawLaHXDGRoo3gSyPX5sjHRbg6Qf6y97scqxpgwUbIL3Z9DqxKnOYevDZYujNB8ow4GsK9EGDCk3wNUSYJEElN9hNtCsULCsF15jilBNmNQS5imEEiggcsIwSsLd4rhOom0fdDtE%2BC%2BR6ou6C8IHG8VQhHoh6KOL%2BE5dGk/xBEkcDXIgkYSgEKgPhAqgwQwI9Ia4IbWwC6QyQ5DPUGEHe6FEF4yAWZM2CaAiFT4GxLOuzQlbWR6KoFEAEyANYEAAA4vpiKqYBwK8TPeKfCgmelgAPWTHBwTVoRAwIbdYICrX9pCAwOQWWEvgCoAcB1gVkDgEzV4B%2BBuAvAVAJwAuKg83ImwbYGpFwg8BSAvwDgFoDpykBZQIAJmoDU4mSAeJmgfiZwF4AAxAa0k2SaQDgCwAYAiAFAI6TuBkAKAwSaMGZJADAAuAuEPgHQCrAAwAG6k0gNEDCC4xOAkk9ycwDYH0hog2gS6F5N4BRhqKBAekAwHyquSsAEIYAG4CKIAw%2BJpALAHqCMDiAZJvAfAEaB6DVJEpWgIIIlQtiCoMp5ALcZxPymnQgwbAjwFgFcm7QWAwU0gFqj0xKAjuqUoqqBQynrAqABgReB2A7Iih6Q08RqfwEEC412AUgGQIIAMpqBXJugJIAYCMAoA8e%2BgPaADFgAYwbJuE%2BgPRSakjA7iDAWUHJMaj4hEp1PLiMBlMCXFLAHQPxrQAnLjAwIoEhgPxPEpcV1pJcaoLUEyAuAcQUwdoEEBxCDBSg5QPQKkHSD4h/pYMgoPiGBnDAKgX0y6L0DmBQyxgNQJGfUDmBwylgCMiaJME8BtA9AeM%2BYCUHhkSB1gCgUSTsHJn6BOA3E0gLxPykCSOAhwVQAAA494lwGavNUjCHA7JLof8MONwCEBFQAENYLwF0nsT1gCkpSbTI4CqTSADU2WYzI0kcAtJIAHSepPWAGTjJFVaOIQwslRg/I9AYgBEBtScB2ZnM7mcAF5lSAXQkkoUCLIkp6BRpwgRhBNOkCuyZp6gEqfNNIAiggwyQYKRxLplqSSpzMyqLOEhCmVWZHMrmcCBtnNx%2BZgsiAB4CskmyV8/4cWVJK1nrAEQyoEYJ9JUm8AlZgNFWaQGZnqzNZXU6WYpOUmcB/wYcpmZpJzk1y5ZZgJuarIlm5y9pCZPwJICAA%3D).